test/Datadog.Trace.ClrProfiler.Managed.Tests/ScopeFactoryTests.cs

```diff
@@ -119,5 +132,79 @@ public void CreateOutboundHttpScope_AlwaysCreatesOneAutomaticInstrumentationScop
                 }
             }
         }
+
+        [Theory]
+        [MemberData(nameof(GetDbCommandScopeData))]
+        public void CreateDbCommandScope_ReturnsNullForExcludedAdoNetTypes(Func<Tracer, Scope> createScopeFunc, string typeName)
+        {
+            // Set up tracer
+            var collection = new NameValueCollection
+            {
+                { ConfigurationKeys.AdoNetExcludeTypes, typeName }
+            };
+            IConfigurationSource source = new NameValueConfigurationSource(collection);
+            var tracerSettings = new TracerSettings(source);
+            var tracer = new Tracer(tracerSettings);
+
+            // Create scope
+            var scope = createScopeFunc(tracer);
+
+            Assert.Null(scope);
+        }
```

Comment:
NIT: your test checks that the span is correctly filtered when setting the configuration key, but not that the *other* ones are *not* filtered. For all we know, maybe every span is indiscriminately filtered once you set the config key.

Reply:
I made some changes in f920370d756dd46de9e01dd7330b119e09070676 . Do you think the tests need any additional changes?